fbw-a32nx/src/systems/systems-host/systems/FWC/PseudoFWC.ts

```diff
@@ -70,20 +77,28 @@ enum FwcAuralWarning {
   None,
   SingleChime,
   Crc,
+  CavalryCharge,
 }
 
 export class PseudoFWC {
-  private readonly sub = this.bus.getSubscriber<A32NXFcuBusEvents & EwdSimvars & StallWarningEvents>();
+  private readonly sub = this.bus.getSubscriber<
+    PseudoFwcSimvars & StallWarningEvents & KeyEvents & A32NXFcuBusEvents
+  >();
+
+  private readonly fwsUpdateThrottler = new UpdateThrottler(125); // has to be > 100 due to pulse nodes
+
+  private keyEventManager: KeyEventManager;
```

Comment:
Should be `private keyEventManager?: KeyEventManager;` as it is undefined after construction.